GG_clone_backend/Gemfile

```diff
@@ -30,6 +30,7 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw] # Use sqlite3 as the database for Active Record
   gem 'sqlite3', '~> 1.4'
+  gem 'rspec-rails', '~> 5.0.0'
```

Comment:
You can add this gem under "development/testing"

Reply:
it is under development test